apps/web/src/components/Feed/Socialize/CommentsModal/FeedEventsCommentsModal.tsx

```diff
@@ -4,7 +4,7 @@ import { graphql, useFragment, usePaginationFragment } from 'react-relay';
 import { FeedEventsCommentsModalFragment$key } from '~/generated/FeedEventsCommentsModalFragment.graphql';
```

Comment:
all components and files related to `getOptimisticUserInfo` refactoring was handled in a separate PR: #1886